### ISSUE TITLE
spingfox swagger ui don't show example value 0

### ISSUE THREAD

**Issue by wade-bigdata** [springfox/springfox#3581]
This issue is same as  #1734 , I can see that issue have been closed，But i am using the latest release version 3.0.0，Found the issue was existed
The param config is 
![image](https://user-images.githubusercontent.com/54099385/94511667-8eb61600-024c-11eb-805f-2fabd1c0cdb0.png)
The ui display result is
![image](https://user-images.githubusercontent.com/54099385/94511697-b0170200-024c-11eb-894c-175a887b5b9b.png)

**Comment by Jouo** [comment]
@wade-bigdata Did you fix it?

**Comment by wade-bigdata** [comment]
> @wade-bigdata Did you fix it?

No,I didn't. I can see the corrected restfull result from chrome network tab,
![image](https://user-images.githubusercontent.com/54099385/95647270-a9b44000-0b00-11eb-953e-783c872ba64c.png),So the issue is in swagger-ui module,Because i‘m not familiar with front-end technology，May be i need take much time to debug it

**Comment by NameNull** [comment]
> > @wade-bigdata Did you fix it?
> 
> No,I didn't. I can see the corrected restfull result from chrome network tab,
> ![image](https://user-images.githubusercontent.com/54099385/95647270-a9b44000-0b00-11eb-953e-783c872ba64c.png)
> So the issue is in swagger-ui module,Because i‘m not familiar with front-end technology，May be i need take much time to debug it

![image](https://user-images.githubusercontent.com/11754843/125760157-af446abb-0f57-4320-aec6-602c2a79d7bb.png)
u didn't set example here; plz fix it
